src/vs/workbench/contrib/terminalContrib/suggest/browser/terminal.suggest.contribution.ts

```diff
@@ -286,6 +287,26 @@ registerTerminalAction({
 	run: (c, accessor) => accessor.get(IPreferencesService).openSettings({ query: terminalSuggestConfigSection })
 });
 
+registerTerminalAction({
+	id: TerminalSuggestCommandId.LearnMore,
+	title: localize2('workbench.action.terminal.learnMore', 'Learn More'),
+	f1: false,
+	precondition: ContextKeyExpr.and(ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated), TerminalContextKeys.focus, TerminalContextKeys.isOpen, TerminalContextKeys.suggestWidgetVisible),
+	menu: {
+		id: MenuId.MenubarTerminalSuggestStatusMenu,
+		group: 'center',
+		order: 1
+	},
+	keybinding: {
+		primary: KeyMod.CtrlCmd | KeyCode.KeyK,
```

Comment:
cmd and ctrl+k is too important to override as it's the main Chord keybinding. How about `ctrl/cmd+/`? I don't think that's used generally in the terminal and it's the physical `?` key.

Reply:
ah, I set this just at random then forgot to change it